sdk/storage/azure-storage-blob/azure/storage/blob/_shared/base_client.py

```diff
@@ -74,6 +75,7 @@ def __init__(
         # type: (...) -> None
         self._location_mode = kwargs.get("_location_mode", LocationMode.PRIMARY)
         self._hosts = kwargs.get("_hosts")
+        self._msrest_xml = kwargs.get('msrest_xml', False)
```

Comment:
We'd want to remove these for now most likely.
Though we could in future look at doing some kind of 'opt-in' flag to opt out of msrest.
Might be best to leave that decision for now until the long-term deprecation story for msrest starts developing. Then we can start planning more realistically.